README.md

```diff
@@ -14,12 +14,25 @@ More information is available on the
 
 ## Installation
 
+Prerequisites
+-------------
+* Have the most recent version of Java (Java 1.8)
+* Have Scala downloaded
+
 Installation requires a recent ScalaTion distribution (>= 1.3) from
 [here](http://cobweb.cs.uga.edu/~jam/scalation.html). Once you have
 ScalaTion, make sure the ``SCALATION_JARS`` environment variable is
 set appropriately before continuing. This can be done individually,
 as seen here:
 
+It is important to note that this is the most important step, if the environment
+variable is not correct, the program will not run properly.
+
+Steps (Terminal):
+* Find where you downloaded your copy of Scalation
+* Use pwd to get the path
+* Copy the path to use in the next steps.
+
```

Comment:
You might want to improve punctuation and formatting for this. See [this link](https://guides.github.com/features/mastering-markdown/) for instructions on how to format text in Markdown.